docs/building-modules/handler.md

```diff
@@ -26,37 +26,31 @@ Let us break it down:
 
 ## Implementation of a module `handler`s
```

Comment:
I wonder if we can actually remove this whole section and just describe the ADR 031 approach. I know @amaurymartiny is in favor of that. At least, I would move it to the bottom of the doc and describe as the legacy or deprecated approach.

Reply:
ADR 031 may basically the whole `handler` paradigm so we may want to stop using the word `handler`

Reply:
@aaronc should we just rename this whole section to `Msg` Services (and the file to `msg-services.md`)?
I did something similar as part of the [first round of the "building modules" doc updates](https://github.com/cosmos/cosmos-sdk/pull/7473) for the old `querier.md` file, renaming it to `query-services.md`.

Reply:
> should we just rename this whole section to Msg Services (and the file to msg-services.md)

I think that's a good idea. Would also propose to slightly the ordering of sections like that:
- Msg Services (basically ADR-031)
- Legacy Msg
  - handler

Reply:
Yes, and link to ADR-031

Reply:
@robert-zaremba already added that, see 2nd paragraph here: https://github.com/cosmos/cosmos-sdk/blob/c904f91d1b79ccc6d431f1336978d497e8a5c062/docs/building-modules/msg-services.md#implementation-of-a-module-msg-service